src/oscar/apps/dashboard/ranges/forms.py

```diff
@@ -31,8 +31,9 @@ class RangeProductForm(forms.Form):
         label=_("File of SKUs or UPCs"), required=False, max_length=255,
         help_text=_('Either comma-separated, or one identifier per line'))
 
-    def __init__(self, range, *args, **kwargs):
+    def __init__(self, range, excluded, *args, **kwargs):
```

Comment:
please make this a kwarts, excluded=True/False, otherwise it's really hard to know in the view what this does.

Reply:
done.

Reply:
Yes, and then you can remove it from the __init__ args.

Reply:
done.